### DISCUSSION TITLE
feature_currency

### DISCUSSION THREAD

**Discussion by ryanhtafesa** [ryanhtafesa/T24S2T1EXTG04-REV2#2] [General]
feature_currency discussion!

**Comment by ryanhtafesa** [comment]
@AlexandraKoshur 
Are you going to add in your navigation into your feature? or wait?
Let me know which way you choose, 
If we do it separately, I will need to do a merge conflict when we merge to development. 
If you leave it out, I won't have to, and I'll just add the methods to navigate after you merge.

**Reply by AlexandraKoshur** [reply]
@ryanhtafesa,
I will leave it out if that makes it easier to merge later.

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa 
The only problem is if I am not adding the navigation, I can't run my Currency Converter, as it's not opening. Do you think I should push it as is?

**Reply by ryanhtafesa** [reply]
@AlexandraKoshur 

The development branch is ready for you to merge your feature_currency into it. 
You were right about the navigation, I needed to add it in to be able to test it and screenshot it. 

What I have done is just some basic navigation - added navigation for mine only, and the exit button to return to the main navigation page. When you pull the latest version, and merge, you will probably get some merge conflicts on Navigation Page.xaml. 
I imagine your code you've entered might be on the same rows as mine... like below

Hopefully you feel comfortable handling the merge conflict? When you get the error, you can choose to keep both lines of code and insert them above or below so we have both our code in the file then resolve it. 

![image](https://github.com/user-attachments/assets/d601bd79-cee3-4ae6-8f9c-d5689e289164)

Once you're done, I'll carry out the last bit and perhaps we can try do the final Section G saturday night or Sunday ?

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa,
 thank you for helping me to resolve the conflict at merge, this is a great team work and I appreciate it a lot! 
Hopefully, we can do this and move to the next section.

**Reply by ryanhtafesa** [reply]
No problems! Thanks for all your help too. 
It's been a challenge!
